psi4/src/psi4/cc/ccresponse/roa.cc

```diff
@@ -139,15 +145,23 @@ void roa() {
             psio_read_entry(PSIF_CC_INFO, lbl1, (char *)tensor0[0], 9 * sizeof(double));
         }
 
-        if (params.wfn == "CC2")
+        std::stringstream tag_tensor0;
+        if (params.wfn == "CC2") {
             outfile->Printf("\n     CC2 Optical Rotation Tensor (Velocity Gauge): %s\n", lbl1);
-        else if (params.wfn == "CCSD")
+            tag_tensor0 << "CC2 OPTICAL ROTATION TENSOR (VEL) @ 0NM";
+        }
+        else if (params.wfn == "CCSD") {
             outfile->Printf("\n    CCSD Optical Rotation Tensor (Velocity Gauge): %s\n", lbl1);
+            tag_tensor0 << "CCSD OPTICAL ROTATION TENSOR (VEL) @ 0NM";
```

Comment:
Just checking that this set twice: `CCSD OPTICAL ROTATION TENSOR (VEL) @ 0NM` https://github.com/psi4/psi4/pull/2022/files#diff-13ff4e13f97ab3afef025c04b5535b0d14561dfdf20e0c22960b5d1656a4394aR194

Reply:
yes, the structure of `roa.cc` isn't optimal in that it goes ahead and re-computes tensors you may already have. at some point we should split this up, but for now if you want to do all three properties it's easiest to just use `roa_tensor` and compute the rotation and polarizabilities from the tensors yourself.